packages/google_maps_flutter/google_maps_flutter_android/README.md

```diff
@@ -70,8 +70,14 @@ AndroidMapRenderer mapRenderer = AndroidMapRenderer.platformDefault;
   }
 ```
 
-Available values are `AndroidMapRenderer.latest`, `AndroidMapRenderer.legacy`, `AndroidMapRenderer.platformDefault`.
-Note that getting the requested renderer as a response is not guaranteed.
+NOTE: `AndroidMapRenderer.platformDefault` corresponds to `AndroidMapRenderer.latest`.
+
+WARNING: `AndroidMapRenderer.legacy` is known to crash apps and is no longer supported.
```

Comment:
it might be valuable to say who is not supporting this mode. Specifically the google maps team is no longer supporting AndroidMapRenderer.legacy and therefor the flutter team will not investigate issues either.

Reply:
Added more context here.